go/sqltypes/value.go

```diff
@@ -46,11 +46,10 @@ type Value struct {
 }
 
 // MakeTrusted makes a new Value based on the type.
-// If val is nil, the Value is NULL. If the value is an integral,
-// then val must be in its cannonical form. This function should
-// only be used if you know the value and type conform to the rules.
-// Every place this function is called, a comment is needed that explains
-// why it's justified.
+// If the value is an integral, then val must be in its cannonical
+// form. This function should only be used if you know the value
+// and type conform to the rules.  Every place this function is
+// called, a comment is needed that explains why it's justified.
```

Comment:
Actually, the rule applies only outside sqltypes :). You'll see that I've used that function all over the place inside the package.
But fair is fair, I'll change the comment to clarify this exception.